cpp/src/Slice/Parser.cpp

```diff
@@ -3458,8 +3458,8 @@ Slice::InterfaceDef::createOperation(
             if (baseName == newName2)
             {
                 ostringstream os;
-                os << "operation `" << name << "' differs only in capitalization from operation" << " `" << op->name()
-                   << "', which is defined in a base interface";
+                os << "operation `" << name << "' differs only in capitalization from operation"
```

Comment:
All the changes outside slice2cs/Gen.cpp and csharp are for the ongoing clang-format issue.